src/components/Home.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import BookList from './BookList';
+
+const Home = () => (
+  <>
+    <BookList />
+  </>
+);
+
+export default Home;
```

Comment:
Since `Home` is just a component which renders only one other component which is `BookList`, I would recommend removing this component and directly using the `BookList` component instead, unless in future you are not planning to add some more components to this component.